src/components/QuotesContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import QuotesText from "./QuotesText";
+import Image from "next/image";
+
+export default function QuotesContainer() {
+  const [quotesText, setQuotesText] = useState(Quotes[0].quote);
+  const [buttonHighlighted, setButtonHighLighted] = useState(0);
+
+  return (
+    <div className="bg-background-quotes flex justify-center p-20">
+      <div className="flex items-center justify-center flex-col text-gray-200 font-serif">
+        <h1 className="text-2xl flex py-6">Don't just take our word for it</h1>
+        <QuotesText
+          Quotes={Quotes}
+          setQuotesText={setQuotesText}
+          buttonHighlighted={buttonHighlighted}
+          setButtonHighLighted={setButtonHighLighted}
+        />
+        <div className="flex flex-col items-center text-center py-20 text-2xl space-y-10">
+          <Image
+            className="flex"
+            src="/quote-icon-white.png"
+            height={21}
+            width={15.75}
+            alt="alt"
+          />
+          {quotesText}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+const Quotes = [
+  {
+    title: "/logo-gear-patrol-white.png",
+    quote:
+      "“Bellroy can take a backbreaker of a wallet and make it smaller than a deck of cards.”",
+    imageDimensions: [164.44, 30],
+  },
+  {
+    title: "/mens-journal-white.avif",
+    quote: "“Seriously slim wallets and passport holders.”",
```

Comment:
both of these quotes are meant to have extra quotes to be displayed?

Reply:
Yes, its apart of the styling of the website.